source/Applications/Advanced/TransformViaArucoMarker/TransformViaArucoMarker.cpp

```diff
@@ -0,0 +1,315 @@
+/*
+This example shows how to transform a point cloud using the pose of an aruco marker.
```

Comment:
Use the same description (or more detailed, thus longer) as on line 59.

Reply:
done